.gitignore

```diff
@@ -30,8 +30,7 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
-/app/assets/builds/*
-!/app/assets/builds/.keep
+
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add your own name logo created by you at the top of the readme file to make your readme attractive and present you as a brand in the software industry 👍 to catch the recruiters' eyes 😊